### ISSUE TITLE
Request: Blank/Copycat/Placeholder/Modifiable Facades

### ISSUE THREAD

**Issue by AvremelM** [AppliedEnergistics/Applied-Energistics-2#8230]
### Describe the feature

**TL;DR:**  A new Facade item that can be retextured *after* being placed.

In my head, this works like Create's "Copycat" blocks, or FramedBlocks, or Dark Utilities' Sneaky Block etc. That is, you would 
1. Craft a Blank Facade (name TBD)
2. Place it anywhere Facades can be placed
3. Right click the placed Blank Facade with a block you want it to look like.
4. Using a wrench once removes the texture, using the wrench again removes the Blank Facade.

Step 3 could theoretically also be replaced by a different retexturing mechanic, e.g. something like the Color Applicator (with a GUI perhaps), or something else. 

### Reasons why it should be considered

Currently, there are some major challenges with facades, including
* Item Registry/JEI spam/lag etc
* * "How do I craft facades" questions when modpack devs hide facades from JEI because of above.
* Having to craft, store, and/or retrieve a bunch of differently textured facades any time you want to hide cables etc
* Probably more that I'm not thinking of

Blank Facades would solve all of that. No item registry spam, no JEI lag/integration hell, no need to search for/craft the right facades every time you need one or fill your inventory with a dozen different facades when building a new setup. Just one item.

### Additional details

Technically, this doesn't *have* to replace the existing Facades, it could just be an additional Facade item with a different/more expensive crafting recipe.

**Comment by Akivas** [comment]
+1 this.
You would also need to be able to chose the "face" you're placing on the Blanc Facade, or change the orientation of the Blanc Facade.
Like with Logs, if you want to place Log Facades on the side for instance.
